content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -77,28 +77,28 @@ The output is similar to this:
 
     apiVersion: v1
     data:
-      .dockerconfigjson: eyJodHRwczovL2luZGV4L ... J0QUl6RTIifX0=
+      .dockercfg: eyJodHRwczovL2luZGV4L ... J0QUl6RTIifX0=
     kind: Secret
     metadata:
       ...
       name: regcred
       ...
-    type: kubernetes.io/dockerconfigjson
+    type: kubernetes.io/dockercfg
 
-The value of the `.dockerconfigjson` field is a base64 representation of your Docker credentials.
+The value of the `.dockercfg` field is a base64 representation of your Docker credentials.
 
-To understand what is in the `.dockerconfigjson` field, convert the secret data to a
+To understand what is in the `.dockercfg` field, convert the secret data to a
 readable format:
 
-    kubectl get secret regcred --output="jsonpath={.data.\.dockerconfigjson}" | base64 -d
+    kubectl get secret regcred --output="jsonpath={.data.\.dockercfg}" | base64 -D
 
 The output is similar to this:
 
     {"auths":{"yourprivateregistry.com":{"username":"janedoe","password":"xxxxxxxxxxx","email":"jdoe@example.com","auth":"c3R...zE2"}}}
 
 To understand what is in the `auth` field, convert the base64-encoded data to a readable format:
 
-    echo "c3R...zE2" | base64 -d
+    echo "c3R...zE2" | base64 -D
```

Comment:
Why change it?

Reply:
`-d` is not a valid flag for the `base64` exec. 

```
$ base64 --help
Usage:  base64 [-hvD] [-b num] [-i in_file] [-o out_file]
  -h, --help     display this message
  -D, --decode   decodes input
  -b, --break    break encoded string into num character lines
  -i, --input    input file (default: "-" for stdin)
  -o, --output   output file (default: "-" for stdout)
```

Reply:
We've seen this before. You should use `--decode` instead of `-d` or `-D` to make sure the command works across different platforms.